addons/mail/models/res_config_settings.py

```diff
@@ -43,3 +45,31 @@ def _compute_fail_counter(self):
             ('date', '>=', previous_date),
             ('state', '=', 'exception'),
         ])
+
+    def open_mail_notification_paynow(self):
+        layout_id = self.env.ref('mail.mail_notification_paynow').id
+        return {
+            'type': 'ir.actions.act_window',
+            'name': 'Mail Layout',
+            'view_mode': 'form',
+            'res_id': layout_id,
+            'res_model': 'ir.ui.view',
+        }
+
+    def open_message_notification_email(self):
+        layout_id = self.env.ref('mail.message_notification_email').id
```

Comment:
If it is removed, this crashes. We should try to be defensive, if somehow this template has been removed, normally chatter still works (just no layouting done). We should go towards being defensive :) .